news/2018-03-18-featured-artist-track-updates.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: post
+title: "Featured Artist Track Updates"
+date: 2018-03-18 05:00:00 +0000
+---
+
+As announced with [Trial & Error's release](https://osu.ppy.sh/home/news/2018-03-06-new-featured-artist-trial-and-error), we're proud to present **thirty** new tracks from for our existing Featured Artists, including [IAHN](https://osu.ppy.sh/beatmaps/artists/3), [yuki.](https://osu.ppy.sh/beatmaps/artists/4), [Helblinde](https://osu.ppy.sh/beatmaps/artists/5), [dark cat](https://osu.ppy.sh/beatmaps/artists/6), [Loki/Thaehan](https://osu.ppy.sh/beatmaps/artists/7), [Rin/Function Phantom](https://osu.ppy.sh/beatmaps/artists/14) and the [Fractal Dreamers](https://osu.ppy.sh/beatmaps/artists/15)!
+
+![](/wiki/shared/news/2018-03-18-featured-artist-track-updates/banner.jpg)
+
+Without further ado, let's get stuck in to the new tracks:
+
+## IAHN
+
+We've got **four** new tracks lined up from IAHN in his classic electronic style. [Check out his Featured Artist listing for the downloads!](https://osu.ppy.sh/beatmaps/artists/3)
+
+### All In One
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/3/previews/385.mp3" type="audio/mpeg">
+</audio>
+
+### Burn Out
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/3/previews/386.mp3" type="audio/mpeg">
+</audio>
+
+### Feel It
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/3/previews/387.mp3" type="audio/mpeg">
+</audio>
+
+### Winter Vacation
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/3/previews/388.mp3" type="audio/mpeg">
+</audio>
+
+## yuki.
+
+**Two** brand new, osu! exclusive tracks arrive with this update from yuki. [Check out his Featured Artist listing for more details and links to the beatmap files for the new tracks!](https://osu.ppy.sh/beatmaps/artists/4)
+
+### daft
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/4/previews/389.mp3" type="audio/mpeg">
+</audio>
+
+### Hello! World (feat. setsunan)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/4/previews/390.mp3" type="audio/mpeg">
+</audio>
+
+## Helblinde
+
+Though we've come to learn that Helblinde has retired from releasing under this alias for now, we're proud to help keep his legacy alive with the release of his last EP *Omnia Vanitas*, sporting **four** new tracks to test your mapping mettle against.
```

Comment:
to help him keep
sporting -> releasing?

Reply:
sporting is like showcasing or showing off, so it works here